boxsdk/object/item.py

```diff
@@ -171,6 +171,11 @@ def get_shared_link(self, access=None, etag=None):
             If specified, instruct the Box API to create the link only if the current version's etag matches.
         :type etag:
             `unicode` or None
+        :param unshared_at:
+            The day that this link should be disabled at. Timestamps are rounded off to the given day.
+            This field can only be set if the user is not a free user.
+        :type unshared_at:
+            `unicode` or None
```

Comment:
This is not Pythonic to take a unicode string. This should take a date or datetime.